wgpu-core/src/command/render.rs

```diff
@@ -745,6 +790,7 @@ impl<'a, A: HalApi> RenderPassInfo<'a, A> {
                 .views
                 .use_extend(&*view_guard, at.view, (), ())
                 .map_err(|_| RenderPassErrorInner::InvalidAttachment(at.view))?;
+            check_multiview(color_view)?;
             add_view(color_view, "color")?;
```

Comment:
we should do this logic in `add_view`, which is already called

Reply:
`check_multiview` is called below on the resolve too, as it has to match, which does not have `add_view`. The function takes a mutable reference to `detected_multiview` and thus can't be mutable borrowed by `add_view` in addition to being used there.